python/ray/_private/accelerators/accelerator.py

```diff
@@ -0,0 +1,123 @@
+from abc import ABC, abstractmethod
+from typing import Optional, List, Tuple
+
+
+class AcceleratorManager(ABC):
+    """This class contains all the functions needed for supporting
+    an accelerator family in Ray."""
+
+    @staticmethod
+    @abstractmethod
+    def get_resource_name() -> str:
+        """Get the name of the resource representing this accelerator family.
+
+        Returns:
+            The resource name: e.g., the resource name for Nvidia GPUs is "GPU"
+        """
```

Comment:
I think `AcceleratorManager` should provide `get_resource_type()` interface,
while `get_resource_name` provide detail resource name, such as NvidiaGPU, IntelGPU
`get_resource_type` provide resource type, such as NvidiaGPU is GPU, IntelGPU is GPU